IntegrationExample/Processing/View/ProcessingView.swift

```diff
@@ -6,17 +6,45 @@
 //
 
 import SwiftUI
+import Combine
 import MimiCoreKit
 
 struct ProcessingView: View {
     
+    @EnvironmentObject var appDelegate: AppDelegate
```

Comment:
Could you inject the `PartnerHeadphoneConnectivityController` instead of AppDelegate?
```suggestion
   @EnvironmentObject var headphoneConnectivityController: PartnerHeadphoneConnectivityController
```